src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6525,7 +6529,9 @@ static void partList(XmlWriter& xml, Score* score, MxmlInstrumentMap& instrMap)
             MxmlReverseInstrumentMap rim;
             initReverseInstrMap(rim, instrMap);
             for (int instNr : rim.keys()) {
-                scoreInstrument(xml, static_cast<int>(idx) + 1, instNr + 1, MScoreTextToMXML::toPlainText(rim.value(instNr)->trackName()));
+                scoreInstrument(xml, static_cast<int>(idx) + 1, instNr + 1, MScoreTextToMXML::toPlainText(rim.value(
+                                                                                                              instNr)->trackName()), rim.value(
+                                    instNr));
```

Comment:
Uncrustify has made this line quite unreadably. Maybe we can help it a bit by manually formatting it?
```suggestion
                scoreInstrument(xml, static_cast<int>(idx) + 1, instNr + 1, 
                                MScoreTextToMXML::toPlainText(rim.value(instNr)->trackName()), 
                                rim.value(instNr));
```

Reply:
Done